File: deaf_grandma.js
function deafGrandma() {
    let goodbyes = 0;
    let lowerReg= /[a-z]/
    let response = window.prompt("HEY KID!");
    
    while(goodbyes!=2){
    
        if(response == "GOODBYE!"){
            goodbyes++;
            if(goodbyes===1){
                response=window.prompt("LEAVING SO SOON?");
            }    
        }
        else if(response===""){
            response=window.prompt("WHAT?!")
        }
        else if(lowerReg.test(response)){
            response=window.prompt("SPEAK UP, KID!");
        }
        else{
            response=window.prompt("NO, NOT SINCE 1946!");
        }
    }
    window.alert("LATER SKATER!")
    

}

deafGrandma();

File: deaf_grandma.py
from urllib import response
import re

def deaf_grandma():

    goodbyes = 0
    response = input("HEY KID!\n")

    while goodbyes != 2:

        if response == "GOODBYE!":
            goodbyes+=1
            if goodbyes == 1:
                response = input("LEAVING SO SOON?\n")
        elif response == "":
            response= input("WHAT?!\n")
        elif(re.search("[a-z]",response)!=None):
            response= input("SPEAK UP, KID!\n")
        else:
            response = input("NO, NOT SINCE 1946!\n")    
    
    print("LATER SKATER!")

deaf_grandma()
